src/bin/js/index_js.mld

```diff
@@ -0,0 +1,7 @@
+{1 Alt-ergo-js}
+
+A Javascript version of the alt-ergo package is available with the command [make js]. The command [make js] will create the javascript file [alt-ergo-js].
+To run this command you will need this aditionnal dependencies [js_of_ocaml-compiler] and [zarith_stubs_js].
```

Comment:
your js_of_ocaml deps seems to be `js_of_ocaml`, `js_of_ocaml-compiler` and `js_of_ocaml-lwt`

Reply:
I've updating this documentation and divided it for my 3 rules and corresponding deps